src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/custom/DSLDoNotDisplayEmptyMethodStrategy.java

```diff
@@ -0,0 +1,62 @@
+package com.adq.jenkins.xmljobtodsl.dsl.strategies.custom;
+
+import com.adq.jenkins.xmljobtodsl.dsl.strategies.*;
+import com.adq.jenkins.xmljobtodsl.parsers.PropertyDescriptor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DSLDoNotDisplayEmptyMethodStrategy extends DSLMethodStrategy {
```

Comment:
Let's leave a comment above the class to kinda explain what we're doing here & why